### PR TITLE
Implement GetNetworkParameters RPC Method

### PR DESCRIPTION
**Pull request by jribbink**
Closes: #232 

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-go-sdk/blob/master/CONTRIBUTING.md#styleguides).
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: access/http/models/model_transaction_result.go [hunk 1]
```diff
@@ -9,10 +9,11 @@
 package models
 
 type TransactionResult struct {
-	BlockId    string                `json:"block_id"`
-	Execution  *TransactionExecution `json:"execution,omitempty"`
-	Status     *TransactionStatus    `json:"status"`
-	StatusCode int32                 `json:"status_code"`
+	BlockId      string                `json:"block_id"`
+	CollectionId string                `json:"collection_id"`
```
Comment (nvdtf) [inline access/http/models/model_transaction_result.go:13]: Is this part of the PR changes? Does it need testing?

Comment (jribbink) [reply access/http/models/model_transaction_result.go:13]: This is just taken from https://github.com/onflow/flow-go/blob/master/engine/access/rest/models/model_transaction_result.go.  I had to copy/paste the REST models & GRPC protobufs from flow-go to update to latest version.

The ones in this SDK were outdated and didn't have the `GetNetworkParameters` method.

Comment (nvdtf) [reply access/http/models/model_transaction_result.go:13]: Seems to me you also have to update this function:
https://github.com/onflow/flow-go-sdk/blob/fb5b4a9931490263ede7c8285f517c58fe259682/access/http/convert.go#L357-L374
`TransactionResult` doesn't have `CollectionId` so should consider the gRPC side as well if we're adding.

Comment (jribbink) [reply access/http/models/model_transaction_result.go:13]: Ah, yeah gotcha.  I'll comb through and deal with all the side effects from updating the REST models.

**Review by chasefleming — APPROVED** [review]

**Pull request merged**